scripts/zkai_hybrid_proof_pressure_selector_gate.py

```
@@ -0,0 +1,736 @@
+    expect_equal(summary.get("stwo_two_proof_frontier_typed_bytes"), EXPECTED_STWO_FRONTIER_TYPED_BYTES, "summary frontier")
+    expect_equal(summary.get("nanozk_paper_reported_bytes"), EXPECTED_NANOZK_REPORTED_BYTES, "summary NANOZK")
+    if not attack_next_routes:
+        raise HybridSelectorError("selector has no attack-next route")
+    if not no_go_now_routes:
+        raise HybridSelectorError("selector has no no-go-now route")
+    if comparable_routes:
+        raise HybridSelectorError("selector proof comparability overclaim")
+    if summary.get("go_gate") != "GO_SELECTOR_HAS_ATTACK_NEXT_AND_NO_GO_NOW_WITH_ZERO_COMPARABLE_ROWS":
+        raise HybridSelectorError("GO gate drift")
+    if not isinstance(payload.get("source_artifacts"), list) or len(payload["source_artifacts"]) != 5:
+        raise HybridSelectorError("source artifact inventory drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)